src/templates/syllabusTemplate.tsx

```diff
@@ -94,7 +94,21 @@ const SECTION_DESCRIPTION: { [key in SectionID]: React.ReactNode } = {
       Feel free to mark some as "skipped" and revisit them at a later time!
     </>
   ),
-  bronze: topicsWarning,
+  bronze: (
+    <>
+      {topicsWarning}
+      <br />
+      If you prefer videos instead of text, check out CPI's free{' '}
```

Comment:
just curious, what's the `{' '}` at the end do

Reply:
adds a space